src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+
+    <link rel="preload" as="script" href="super-important.js">
+    <link rel="preload" as="style" href="critical.css">
+
+Most of this is probably what you’d expect, except perhaps for the “as”
+attribute. This allows you to tell the browser the type of the resource you’re
+loading, so that it can be handled correctly; in fact, the browser will not use
+the preloaded resource unless the correct type is set. The resource will be
+loaded with the same priority as it would otherwise, but now the browser knows
+about it ahead of time, allowing for the download to start earlier.
+
+Note that `<link rel="preload">` is a compulsory instruction to the browser;
+unlike the other resource hints we’ll be talking about, it’s something the
+browser must do, rather than merely an optional hint. This makes it particularly
+important to test carefully, to insure that you’re not accidentally causing
+anything to fetch twice by using it, or fetching something that’s not needed.
+
+Resources that are fetched using `<link rel="preload">`, but not used by the
+current page within 3 seconds will trigger a warning in the Console in Chrome
+Developer Tools, so be sure to keep an eye out for these!
+
+<figure>
+  <div class="aspect-ratio"
+       style="width: 1050px; --aspect-ratio-w: 1050; --aspect-ratio-h: 244">
+    <img src="images/res-prio-timeout.png"
+    alt="An example of a preload timeout error in Chrome Developer Tools">
+  </div>
+</figure>
+
+### Use-case: Fonts
+
+Fonts are a great example of late-discovered resources that must be fetched,
+often sitting at the bottom of one of several CSS files loaded by a page.
+
+In order to reduce the amount of time the user has to wait for the text content
+of your site, as well as avoid jarring flashes between system fonts and your
+preferred ones, you can use `<link rel="preload">` in your HTML to let the
+browser know immediately that a font will be needed.
+
+    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+
+Note that the use of `crossorigin` here is important; without this attribute,
+the preloaded font will be ignored by the browser, and a new fetch will take
+place. This is because fonts are expected to be fetched anonymously by the
+browser, and the preload request is only made anonymous by using the
+`crossorigin` attribute.
+
+Caution: If you’re using a CDN, such as Google Fonts, be sure that the font files
+you’re preloading match the ones in the CSS, which can be tricky due to unicode
+ranges, weights, and font variants. Fonts can also be regularly updated, and if
+you’re preloading an old version while using the CSS for a newer one, you may
+end up downloading two versions of the same font and wasting your users’
+bandwidth. Consider using `<link rel="preconnect">` instead for easier
+maintenance.
+
+### Use-case: Critical Path CSS and JavaScript
+
+When talking about page performance, one useful concept is that of the
```

Comment:
Lean this a bit: "... one useful concept is the "critical path". The critical path refers to the resources that must be loaded before your initial render. These resources are critical to getting the first pixels on the screen, for example CSS.

Reply:
Done, although I prefer the "like CSS" bit earlier, between commas. Let me know if that doesn't work.